templates/glance/config/glance-api-config.json

```diff
@@ -2,28 +2,31 @@
     "command": "/usr/bin/glance-api --config-dir /etc/glance/glance.conf.d",
     "config_files": [
       {
-        "source": "/var/lib/config-data/merged/glance.conf.d",
-        "dest": "/etc/glance/glance.conf.d",
+        "source": "/var/lib/config-data/default/00-config.conf",
+        "dest": "/etc/glance/glance.conf.d/00-config.conf",
         "owner": "glance",
-        "perm": "0750"
+        "perm": "0600"
       },
       {
-        "source": "/var/lib/config-data/merged/glance-image-import.conf",
-        "dest": "/etc/glance/glance.conf.d/glance-image-import.conf",
+        "source": "/var/lib/config-data/default/02-config.conf",
```

Comment:
have we missed 01-config.conf?

Reply:
it's basically 01 in the umbrella glance, but 02 in the glance-api service, because the same config is passed to the instances created by the top-level CR [1].
We can improve that in a follow up change, or rename 02-config to 01-config: the goal is keeping consistency with the naming across the board, and don't expected a change in their meaning.
I noted this as a potential follow up! How does that sound?

[1] https://github.com/openstack-k8s-operators/glance-operator/blob/main/controllers/glance_controller.go#L711